utility/mm2_upgrade-install-repo

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/python
+
+"""
+This script changes the directory path from the fedora-install-X repositories
+for a given version.
+
+This can be used several time during the release cycle to move a repo from
+a beta state to a release state.
+
+This is currently kind of a work-around until the logic can be added into
+the UMDL script.
+
+TODO: test IRL
+"""
+
+import sys
+import re
+import os
+
+sys.path.insert(0, os.path.join(os.path.dirname(
+    os.path.abspath(__file__)), '..'))
+import mirrormanager2.lib
+from mirrormanager2.lib.model import Repository
+from optparse import OptionParser
+
+
+# moving from pub/fedora/linux/releases/test/22_Beta/Server/x86_64/os/
+# to          pub/fedora/linux/releases/22/Server/x86_64/os
+# TODO: adjust the UMDL to do this
+
+
+def move_install_repo(session, version, test=False, debug=False):
+    product=mirrormanager2.lib.get_product_by_name(session, "Fedora")
+    ver = mirrormanager2.lib.get_version_by_name_version(
+            session, product.name, version)
+    if not ver:
+        print 'No version found for %s-%s' % (product.name, version)
+        return
+
+    for a in mirrormanager2.lib.get_arches(session):
+        if a.name == u'source':
+            continue
+
+        if options.version == 'development':
+            # We do not change development repos
+            continue
+        else:
+            prefix=u'fedora-install-%s' % ver.name
+            if a.primary_arch:
+                d = u'pub/fedora/linux/releases/%s/Server/%s/os' % (
+                    ver.name, a.name)
+                category = mirrormanager2.lib.get_category_by_name(
+                    session, 'Fedora Linux')
+            else:
+                d = u'pub/fedora-secondary/releases/%s/Server/%s/os' % (
+                    ver.name, a.name)
+                category = mirrormanager2.lib.get_category_by_name(
+                    session, 'Fedora Secondary Arches')
+
+            if not test:
+                if not os.path.isdir(os.path.join('/srv', d)):
+                    print "directory %s does not exist on disk, skipping "\
+                        "creation of a repository there" % d
+                    continue
+
+            dobj = mirrormanager2.lib.get_directory_by_name(session, d)
+
+            if not dobj:
+                print "directory %s exists on disk, but not in the database"\
+                    " yet, skipping creation of a repository there until "\
+                    "after the next UMDL run." % d
+                continue
+
+            repo = mirrormanager2.lib.get_repo_prefix_arch(
+                session, prefix, a.name)
+            if not repo:
+                print 'No repo found for prefix: %s - arch: %s' % (
+                    prefix, a.name)
+                return
```

Comment:
Def yes, before it was returning a list but now it return a single element so it could continue